pkg/multimodelconfig/configmap.go

```diff
@@ -0,0 +1,110 @@
+package multimodelconfig
+
+import (
+	"encoding/json"
+	"k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var logger = log.Log.WithName("MultiModelConfig")
+
+type ConfigsDelta struct {
+	updatedCfg map[string]string
+	deletedCfg map[string]string
+}
+
+type ModelConfig struct {
+	fileName    string
+	fileContent *ModelDefinition
+}
+
+type ModelDefinition struct {
```

Comment:
Should this be the same as the TrainedModel struct?

Reply:
We will replace ModelDefinition with ModelSpec in TrainModel after https://github.com/kubeflow/kfserving/pull/991 is merged to master. I add a comment here.

Reply:
I recommend just waiting for that to submit first, or stacking your PR on top of #991

Reply:
@ellistarn and @wengyao04 I submitted https://github.com/kubeflow/kfserving/pull/1009 to merge ModelSpec into master branch.